client/app/screens/CreateMatchScreen.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect } from 'react';
+import { Button, ButtonText, ScrollView, FormControl, FormControlLabel, FormControlLabelText, Input, InputField, View, Text} from "@gluestack-ui/themed";
+import { AppStackParamList } from 'app/navigators/AppNavigator';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { colors, typography} from "app/theme"
+
+type CreateMatchScreenNavigationProp = NativeStackNavigationProp<
+  AppStackParamList,
+  'Welcome'
+>
+
+export interface CreateMatchScreenProps {
+  navigation: CreateMatchScreenNavigationProp;
+}
+
+/**
+ * CreateMatchScreen component allows users to create and send new match.
+ * @component
+ */
+export function CreateMatchScreen(props: CreateMatchScreenProps) {
+  const [leagueName, setLeagueName] = React.useState<string>("");
+  const [week, setWeek] = React.useState<string>("");
+  const [teamName, setTeamName] = React.useState<string>("");
+  const [createMatchDisabled, setCreateMatchDisabled] = React.useState<boolean>(true);
+  const NFL_WEEKS = 18;
+
+  // TODO: Allow matches for current/future weeks only. 
+  // Pass the current week as a prop/state to this screen.
+  // Output error message if week is invalid.
```

Comment:
I think when we do this we should still include the week's input but have the input disabled and display the week. This way it will be extra clear to the user what week the match will be.

Reply:
Yes, I was also thinking something along the same lines. In the future, we could have an API request that gets the current week, which we can use here. 

When we implement this in the future, should we have it so that people can make matches for future weeks too? Or just make matches for the current week?

Reply:
Yes, we could either have it where they navigate to a future week in the match list (and it autofills that future week) or they can edit week in the form.